src/module-elasticsuite-core/Model/Healthcheck/SearchEngineConfigCheck.php

```diff
@@ -63,7 +64,7 @@ public function __construct(
         UrlInterface $urlBuilder,
         int $sortOrder = 50
     ) {
-        parent::__construct($urlBuilder, $sortOrder);
+        parent::__construct($urlBuilder, $sortOrder, MessageInterface::SEVERITY_CRITICAL);
```

Comment:
Please add the severity is as an exposed constructor parameter, after the sortOrder.